apps/runtime-demo/host/src/index.ts

```diff
@@ -0,0 +1 @@
+import('./bootstrap');
```

Comment:
Wont be needed soon ;) if we point this at enhanced and use async boundary - however i think we can improve how async bounady is applied once we copy over fork from MFP again - low priority right now. Future nice to have